images/macos/provision/core/openjdk.sh

```diff
@@ -26,6 +26,7 @@ createEnvironmentVariable() {
         JAVA_VERSION="1.${JAVA_VERSION}"
     fi
     local JAVA_PATH=$(/usr/libexec/java_home -v${JAVA_VERSION})
+    export ${VARIABLE_NAME}=${JAVA_PATH}
```

Comment:
Why do we need this line? Please remove it

Reply:
it is required since without it tests will fail on "JAVA_PATH" verification

Reply:
I think we should add `source $HOME/.bashrc` line as a part of `invoke_tests` function to reload the whole environment instead of setting specific variables.